lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss

```diff
@@ -1,14 +1,72 @@
 @charset "UTF-8";
 
+
+// For all templates.
+.views-view-grid {
+  .flex-container {
+    @include grid-media-max('md') {
+      display: block;
+
+      .flex-4-of-12 {
+        max-width: 100%;
+      }
+    }
+  }
+}
+
+// News Cards added in page as a paragraph.
+.ptype-news-views {
```

Comment:
The view on the news node page is not picking up these styles as this selector is locked to the paragraph type. Could we target the view instead of the paragraph type here?

Reply:
These styles were just for the card when in a paragraph type.  So that if 1, 2, or 3 are added they display centered and 980px for 1 card or 50% with gutters for 2 or 30% with gutters for 3.

Reply:
We still want all of that for the paragraph type but we also want to avoid this:

![Screen Shot 2020-04-30 at 2 50 38 PM](https://user-images.githubusercontent.com/550602/80762751-0216f880-8af2-11ea-96da-868981069f74.png)

Reply:
Oo. Good point. On it! :)

Reply:
I fixed this.